rfc-018-our-decision-making-process-is-public.md

```diff
@@ -0,0 +1,69 @@
+# Our decision making process is public
+
+## Summary
+
+Our process of making decisions and the decisions themselves MAY be read by
+all staff and members of the public.
+
+This topic has previously been [debated as an issue](https://github.com/dxw/tech-team-rfcs/issues/2)
+on this repository. The decision to take no action should still be persisted
+as a formal decision so has become an RFC.
+
+## Problem
+
+This repository started as public by default but do we agree that this is right
+in this case?
+
+Reasons for:
+
+- dxw values include being honest and we agree with the
+[principle of making things open as it makes things better](https://www.gov.uk/guidance/government-design-principles#make-things-open-it-makes-things-better)
+- [the dxw playbook is public](http://playbook.dxw.com) and has not caused us
+concern or prevented us from iterating it
+- others can learn from us and spend more their time making things better for
+users
+- it allows us to share our thinking and rationale on our technical decisions,
+doing so publicly as well as privately can increase trust and empathy with more
+people
+- with the potential for the world watching the bar is raised
+  being open from the start ensures that there is less risk to us should a
+  future team decide to make it public
+- most of the application code we write lives in open source repositories where
+our work and decision making process can already be viewed publicly
+
+Reasons against:
+
+- parts of our discussion make have to include commercially sensitive
+ information eg. incident reports
+- as the public can read the team may feel too uncomfortable to contribute to
+ discussions making it harder to talk with candour
```

Comment:
There is an additional one in that, deciding to make this private at a later date will likely be in response to something happening that we consider to be too sensitive to discuss in public. Therefore, making this repository private is in itself information leaking.

Reply:
I'm afraid I don't follow.

Reply:
This repository is public. If it becomes private (and therefore disappears for other users), someone could infer that we had something happen that meant we needed to make this repository private to discuss it.

Reply:
Thanks for re-explaining. I can see that making it private in future could leak the information to the public that there was now content we deemed sensitive in it. The negative outcome for this could be that the private repo could then become a target for people to glean information from?

I've added a point to that affect but if I've missed the mark again I'm happy to update it.